mlir/lib/Dialect/LLVMIR/IR/LLVMAttrs.cpp

```diff
@@ -43,8 +55,9 @@ void LLVMDialect::registerAttributes() {
 
 bool DINodeAttr::classof(Attribute attr) {
   return llvm::isa<DIBasicTypeAttr, DICompileUnitAttr, DICompositeTypeAttr,
-                   DIDerivedTypeAttr, DIFileAttr, DILabelAttr,
-                   DILexicalBlockAttr, DILexicalBlockFileAttr,
+                   DIDerivedTypeAttr, DIExpressionAttr, DIFileAttr,
+                   DIGlobalVariableAttr, DIGlobalVariableExpressionAttr,
+                   DILabelAttr, DILexicalBlockAttr, DILexicalBlockFileAttr,
```

Comment:
```suggestion
                   DIDerivedTypeAttr, DIFileAttr,
                   DIGlobalVariableAttr,
                   DILabelAttr, DILexicalBlockAttr, DILexicalBlockFileAttr,
```
If the attributes do not derive from DINode anymore, following the LLVM class diagram, we can remove them here.